lib/src/components/NxTransferListHalf/NxTransferListHalf.scss

```diff
@@ -115,12 +115,14 @@ $-visible-item-count: 10;
     padding-left: var(--nx-spacing-2x);
 
     &:hover {
+      background-color: var(--nx-color-interactive-background-hover);
```

Comment:
It looks like on hover & click states, the text colour changes to nx-grey-50 as well.